### PR TITLE
ifconfig, lua, mkdir, pwd: add French translation

### PR DESCRIPTION
**Pull request by ivorysoap**
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/common/ifconfig.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# ifconfig
+
+> Configurateur des interfaces réseau.
+
+- Affiche les paramètres de réseau d'un adaptateur ethernet :
+
+`ifconfig eth0`
```
Comment (navarroaxel) [inline pages.fr/common/ifconfig.md:7]: This is a bug in the English page, the `eth0` should be between `{{}}`:
```suggestion
`ifconfig {{eth0}}`
```
You can add the fix here if you want.

File: pages.fr/common/ifconfig.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Affiche les détails de toutes les interfaces, y compris les interfaces 
+désactivées :
+
+`ifconfig -a`
+
+- Désactive l'interface eth0 :
+
+`ifconfig eth0 down`
```
Comment (navarroaxel) [inline pages.fr/common/ifconfig.md:16]:
```suggestion
`ifconfig {{eth0}} down`
```

File: pages.fr/common/ifconfig.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Active l'interface eth0 :
+
+`ifconfig eth0 up`
```
Comment (navarroaxel) [inline pages.fr/common/ifconfig.md:20]:
```suggestion
`ifconfig {{eth0}} up`
```

File: pages.fr/common/ifconfig.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Assigne une addresse IP à l'interface eth0 :
+
+`ifconfig eth0 {{addresse_ip}}`
```
Comment (navarroaxel) [inline pages.fr/common/ifconfig.md:24]:
```suggestion
`ifconfig {{eth0}} {{addresse_ip}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @ivorysoap, I left some comments.

**Comment by ivorysoap** [comment]
Sure, thanks for that.  Made the edits.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**